mira/metamodel/templates.py

```diff
@@ -340,6 +344,17 @@ def with_context(self, **context) -> "GroupedControlledConversion":
             provenance=self.provenance,
         )
 
+    def get_key(self, config: Optional[Config] = None):
+        return (
+            self.type,
+            *tuple(
+                c.get_key(config=config)
+                for c in sorted(self.controllers, key=lambda c: c.get_curie())
```

Comment:
shouldn't the sort key also be `lambda c: c.get_key(config=config)`? or are these not guaranteed to be comparable?

Reply:
Fixed in https://github.com/indralab/mira/pull/54/commits/d50f20f9f3426a034cba15ecbe65a3ac529bb911